File: Modul-1/Soal 10/PRAK110-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

double hitung_akar_kuadrat(double angka) {
    double hasil;
    double temp;

    hasil = angka / 2; 
    temp = 0;

    while (hasil != temp) {
        temp = hasil;
        hasil = (angka / temp + temp) / 2; 
    }

    return hasil;
}

int main() {
    int alas = 5;
    int tinggi = 12;
    double sisi_miring = hitung_akar_kuadrat(alas * alas + tinggi * tinggi);

    int keliling = alas + tinggi + (int)sisi_miring; 
    int luas = 0.5 * alas * tinggi; 

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", alas);
    printf("Sisi B = %d cm\n", tinggi);
    printf("Sisi C = %d cm\n", (int)sisi_miring); 
    printf("Keliling = %d cm\n", keliling); 
    printf("Luas = %d cm\n", luas); 

    return 0;
}

File: Modul-1/Soal 10/PRAK110-2410817210014-ZAVIER GHANI.py
def cetak_pesan():
    
    alas = 5
    tinggi = 12

    sisi_miring = (alas**2 + tinggi**2) ** 0.5  

    keliling = alas + tinggi + sisi_miring
    luas = 0.5 * alas * tinggi

    print("Diketahui :")
    print(f"Alas = {alas} cm")
    print(f"Tinggi = {tinggi} cm")
    print("Jawab :")
    print(f"Sisi A = {alas} cm")
    print(f"Sisi B = {tinggi} cm")
    print(f"Sisi C = {sisi_miring:.2f} cm")
    print(f"Keliling = {keliling:.2f} cm")
    print(f"Luas = {luas:.2f} cm")

cetak_pesan()
